modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentFolderResourceImpl.java

```diff
@@ -165,24 +175,14 @@ public Page<DocumentFolder> getSiteDocumentFoldersPage(
 	}
 
 	@Override
-	public DocumentFolder patchDocumentFolder(
```

Comment:
I don't find the old PATCH method, did you decided to remove it?

Reply:
We check it together and it seemed to us that was just doing the same logic as the autogenerated logic. That's why we decided to remove it. But we might miss something... Do you have more information about why this patch method was necessary?

Reply:
No, it is true that it seems it is doing the same logic in the end. Well done guys!